src/mmw/bundle.sh

```diff
@@ -113,6 +118,7 @@ done
 VAGRANT_COMMAND="cd /opt/app && \
     $ENSURE_DIRS_EXIST && { \
     $COPY_IMAGES_COMMAND &
+    $COPY_SIMPLE_APP_DATA_COMMAND &
```

Comment:
That command does not appear to do anything.  I delete /var/www/.../model.csv on the app vm, ran that command, and model.csv has not returned.